DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/ViewDelegate/_DelegateCourseInfoCard.cshtml

```diff
@@ -0,0 +1,132 @@
+            Complete by
+          </dt>
+          <partial name="_SummaryFieldValue" model="@Model.CompleteBy" />
+          <dd class="nhsuk-summary-list__actions">
+            <a href="#">Edit<span class="nhsuk-u-visually-hidden"> complete by date</span></a>
+          </dd>
+        </div>
+
+        <div class="nhsuk-summary-list__row details-list-with-button__row">
+          <dt class="nhsuk-summary-list__key">
+            Last updated
+          </dt>
+          <partial name="_SummaryFieldValue" model="@Model.LastUpdated" />
+        </div>
+
+        <div class="nhsuk-summary-list__row details-list-with-button__row">
+          <dt class="nhsuk-summary-list__key">
+            Completed
+          </dt>
+          <partial name="_SummaryFieldValue" model="@Model.Completed" />
+          @if (Model.Completed == null) {
+            <dd class="nhsuk-summary-list__actions">
+              <a href="#">Set completed date</a>
+            </dd>
+          }
+        </div>
+
+        <div class="nhsuk-summary-list__row details-list-with-button__row">
+          <dt class="nhsuk-summary-list__key">
+            Evaluated
+          </dt>
+          <partial name="_SummaryFieldValue" model="@Model.Evaluated" />
+        </div>
+
+        <div class="nhsuk-summary-list__row details-list-with-button__row">
+          <dt class="nhsuk-summary-list__key">
+            Enrollment method
+          </dt>
+          <partial name="_SummaryFieldValue" model="@Model.EnrollmentMethod" />
+        </div>
+
+        <div class="nhsuk-summary-list__row details-list-with-button__row">
+          <dt class="nhsuk-summary-list__key">
+            Logins
+          </dt>
+          <partial name="_SummaryFieldValue" model="@Model.LoginCount.ToString()" />
+        </div>
+
+        <div class="nhsuk-summary-list__row details-list-with-button__row">
+          <dt class="nhsuk-summary-list__key">
+            Learning time
+          </dt>
+          <partial name="_SummaryFieldValue" model="@Model.LearningTime" />
+        </div>
+
+        <div class="nhsuk-summary-list__row details-list-with-button__row">
+          <dt class="nhsuk-summary-list__key">
+            Diagnostic score
+          </dt>
+          <partial name="_SummaryFieldValue" model="@Model.DiagnosticScore.ToString()" />
+        </div>
+
+        @if (Model.IsAssessed) {
+          <div class="nhsuk-summary-list__row details-list-with-button__row">
+            <dt class="nhsuk-summary-list__key">
+              Assessments passed
+            </dt>
+            <partial name="_SummaryFieldValue" model="@Model.TotalAttempts.ToString()" />
+          </div>
+
+          <div class="nhsuk-summary-list__row details-list-with-button__row">
+            <dt class="nhsuk-summary-list__key">
+              Assessment attempts
+            </dt>
+            <partial name="_SummaryFieldValue" model="@Model.AttemptsPassed.ToString()" />
+          </div>

```

Comment:
I think the model values displayed for these two are mixed up.

Reply:
Whoops 🤭